modules/web/src/app/dynamic/enterprise/cluster-backups/list/backups/template.html

```diff
@@ -248,8 +248,8 @@
           *matHeaderRowDef="columns"></tr>
       <tr mat-row
           *matRowDef="let element; columns: columns"
-          class="km-mat-row km-pointer"
-          [ngClass]="{'km-selected': checkSelected(element.id)}"
+          class="km-mat-row"
+          [ngClass]="{'km-selected': checkSelected(element.id), 'km-pointer': element.spec.includedNamespaces}"
```

Comment:
Why do we need this class on whole row? Can you please share a screenshot for this change? Same question for changes in restores and schedules table.

Reply:
the change only cause we show namespaces if not all namespaces is included, so if no namespaces is shown there is no need for km-pointer